src/abstractdataframe/abstractdataframe.jl

```diff
@@ -1280,3 +1283,168 @@ julia> ncol(df)
 
 """
 (nrow, ncol)
+
+"""
+    disallowmissing(df::AbstractDataFrame,
+        cols::Union{ColumnIndex, AbstractVector, Regex, Not, Colon}=:)
+
+Create a new `DataFrame` in which columns `cols` of data frame `df` are converted
+from element type `Union{T, Missing}` to `T` to drop support for missing values.
+
+If `cols` is omitted all columns in the data frame are converted.
+
+**Examples**
+
+```jldoctest
+julia> df = DataFrame(a=Union{Int,Missing}[1,2])
+2×1 DataFrame
+│ Row │ a      │
+│     │ Int64⍰ │
+├─────┼────────┤
+│ 1   │ 1      │
+│ 2   │ 2      │
+
+julia> disallowmissing(df)
+2×1 DataFrame
+│ Row │ a     │
+│     │ Int64 │
+├─────┼───────┤
+│ 1   │ 1     │
+│ 2   │ 2     │
+```
+"""
+function Missings.disallowmissing(df::AbstractDataFrame,
+                         cols::Union{ColumnIndex, AbstractVector, Regex, Not, Colon}=:)
+    idxcols = Set(index(df)[cols])
```

Comment:
Why not this?
```suggestion
    idxcols = unique!(index(df)[cols])
```

Reply:
`Set` will be faster later (we repeatedly do `i in idxcols` check) and `unique!` will fail if `cols` is a `ColumnIndex` (a single column selector not wrapped in a vector)